src/action/macos/create_fstab_entry.rs

```diff
@@ -289,6 +302,10 @@ pub enum CreateFstabEntryError {
     ExistingNixInstallerEntryDisappeared,
     #[error("The `/etc/fstab` entry (previously created by the official install scripts) detected during planning disappeared between planning and executing. Cannot update `/etc/fstab` as planned")]
     ExistingForeignEntryDisappeared,
+    #[error("Unable to determine how to add APFS volume `{0}` the `/etc/fstab` line, likely the volume is not yet created or there is some synchronization issue, please report this")]
+    CannotDetermineUuid(String),
+    #[error("Unable to reliably determine which `/etc/fstab` line to remove, the volume is likely already deleted, the line involving `/nix` in `/etc/fstab` should be removed manually")]
+    EntryNoLongerDeterminable,
```

Comment:
Nit:

```suggestion
    #[error("Unable to reliably determine which `/etc/fstab` line to remove. The line involving `/nix` in `/etc/fstab` should be removed manually")]
    CannotDetermineFstabLine,
```

Reply:
I do think explaining why we cannot determine the fstab line is useful. :thinking:

Reply:
It might change how they percieve the error. Just saying we can't determine it doesn't instill confidence in the user, explaining we cannot because the volume is already deleted assures them both that there isn't some spooky lost volume and helps them understand the error is actually kind of a good thing.

I will change the enum :)